src/Element.js

```diff
@@ -136,6 +137,86 @@ export default class Element extends ParentNode {
         });
         return _array;
     }
+
+    /**
+     * Returns the first element that is a descendant of the element on which it is invoked that matches the
+     * specified group of selectors.
+     * @param {String} query CSS query for selecting element
+     * @return {Element} reference to an Element, or null if element was not found
+     */
+    querySelector(query) {
+        let parser = new CssSelectorParser.CssSelectorParser();
```

Comment:
Good idea, didn't think this way. Seems wished to add just method. I think it would be refactored any way when i will need support of **querySelectorAll** method. Just did it fast for HTMLTable element (which was a goal).